### PR TITLE
[CENNSO-1606] Rf: copy some PS-Information into the following requests

### PR DESCRIPTION
**Pull request by RoadRunnr**
copy some PS-Information into the following requests and encode them in the TP-Previous-PS-Information vendor AVP.
This helps out legacy (v1/v2 ) CDF to include the correct information in the CDR.

WARNING: The TS 32.299 dictionary does not make the Service-Information group extendable.
         Adding the vendor AVP TP-Previous-PS-Information to that group is therefore not backward
         compatible. Use Diameter AVP filters to strip that new AVP when integration with
         an CDF without support for that AVP is needed.

### CODE REVIEW

File: dia/diameter_travelping.dia [hunk 1]
```diff
@@ -6,15 +6,37 @@
 ;; 2 of the License, or (at your option) any later version.
 
 ;; It contains custom Travelping AVPs related to NAT-binding
+;; When adding AVPs make sure to use values not alrady used by the
+;; Travelping RADIUS dictionary:
+;;   https://github.com/travelping/eradius/blob/master/priv/dictionaries/dictionary.travelping
```
Comment (vkatsuba) [inline dia/diameter_travelping.dia:11]: Maybe make sense to use latest tag of eradius instead of using master  link

File: src/ergw_aaa_rf.erl [hunk 2]
```diff
@@ -316,6 +318,17 @@ to_session(_, 'Acct-Interim-Interval', [Interim], {Session, Events})
 to_session(_, _, _, SessEv) ->
     SessEv.
 
+save_prev_request_session_info(Session) ->
+    PrevSessInfoKeys =
+	['QoS-Information',
+	 '3GPP-SGSN-Address', '3GPP-SGSN-IPv6-Address',
```
Comment (vkatsuba) [inline src/ergw_aaa_rf.erl:324]:
```suggestion
	 '3GPP-SGSN-Address',
    '3GPP-SGSN-IPv6-Address',
```

File: dia/diameter_travelping.dia (hunk 1 above)
Comment (javiermtorres) [reply dia/diameter_travelping.dia]: Ok

File: src/ergw_aaa_rf.erl (hunk 2 above)
Comment (javiermtorres) [reply src/ergw_aaa_rf.erl]: ok

**Review by javiermtorres — APPROVED** [review]

**Pull request merged**